### PR TITLE
Correct `algorithem` to `algorithm`

### PR DESCRIPTION
**Pull request by hanyuone**

### CODE REVIEW

**Comment by bararchy** [comment]
@Qwerp-Derp Thanks for the fixes, it seems to fail CI check: 
```crystal
Error in line 1: while requiring "./spec/network_spec.cr"

in spec/network_spec.cr:70: undefined method 'train_algorithem' for Fann::Network::Cascade (did you mean 'train_algorithm'?)

    ann.train_algorithem(LibFANN::TrainEnum::TrainRprop)

        ^~~~~~~~~~~~~~~~


```
I think you missed a change in the specs

**Comment by hanyuone** [comment]
@bararchy Thanks for the catch! Will fix now.

**Comment by hanyuone** [comment]
Ready to PR.

File: src/crystal-fann/standard_network.cr [hunk 1]
```diff
@@ -46,7 +46,7 @@ module Fann
         LibFANN.set_activation_function_output(@nn, func)
       end
 
-      def randomzie_weights(min : Float64, max : Float64)
+      def randomie_weights(min : Float64, max : Float64)
```
Comment (bararchy) [inline src/crystal-fann/standard_network.cr]: @Qwerp-Derp Typo here, radnomie should be randomize

**Comment by bararchy** [comment]
@Qwerp-Derp awesome ! All tests pass , merging

**Pull request merged**